.github/workflows/acceptance-tests-runner.yml

```diff
@@ -455,7 +482,21 @@ jobs:
           AZURE_ATLAS_APP_ID: ${{ inputs.azure_atlas_app_id }}
           AZURE_SERVICE_PRINCIPAL_ID: ${{ inputs.azure_service_principal_id }}
           AZURE_TENANT_ID: ${{ inputs.azure_tenant_id }}
-          TEST_REGEX: "^TestAccConfig"
+          ACCTEST_PACKAGES: |
+            ./internal/service/alertconfiguration
+            ./internal/service/atlasuser
+            ./internal/service/databaseuser
+            ./internal/service/projectapikey
```

Comment:
like how each test group now defines explicitly which resources are included without relying on the naming indirection